packages/x-components/static-docs/experience-search-and-discovery/history-queries.md

```diff
@@ -51,9 +51,14 @@ session will display.
 Combine History Queries with the [Search module](/develop-empathy-platform/ui-reference/components/search/) to ease shoppers' frustrations. That way, you can
 configure it to display only successful history searches.
 
+::: note Search history tracking
+
+Use the [My History](my-history.md) component to control search history tranking. Shoppers can view a complete list of history queries and decide to stop saving search history. If search history is disabled, the History Queries list is cleared.
```

Comment:
```suggestion
Use the [My History](my-history.md) component to control search history tracking. Shoppers can view a complete list of history queries and decide to stop saving search history. If search history is disabled, the History Queries list is cleared.
```

Reply:
I corrected the typo, but i don't know if i would use `tracking` as suggest something like we are sending this to and endpoint...you know?

Maybe this is better:

```suggestion
Use the [My History](my-history.md) component to access and control search history. Shoppers can view a complete list of history queries and decide to stop saving search history. If search history is disabled, the History Queries list is cleared.
```